newzealidar/process.py

```diff
@@ -485,27 +547,34 @@ def main(
             return
 
     lidar_extent_file = (
-        pathlib.Path(utils.get_env_variable("DATA_DIR"))
+        pathlib.Path(env_var.get_env_variable("DATA_DIR"))
         / pathlib.Path("gpkg")
         / pathlib.Path("lidar_extent.gpkg")
     )
-    if lidar_extent_file.exists():
+    # Retrieve the value of the environment variable "USE_AWS_S3_BUCKET"
+    use_aws_s3_bucket = env_var.get_env_variable("USE_AWS_S3_BUCKET", cast_to=bool, allow_empty=True, default=False)
+    s3_manager = S3Manager()
+    s3_objects = s3_manager.list_objects()
```

Comment:
Open conversation from previous PR regarding errors with env_vars and s3_manager. I am working on it. https://github.com/LukeParky/NewZeaLiDAR/pull/2#discussion_r1663527020

Reply:
Fixed